packages/schematics/migrations/legacy-migrations/20180515-switch-to-nx6.ts

```diff
@@ -20,7 +20,7 @@ export default {
     }
 
     updateJsonFile('package.json', json => {
-      json.devDependencies['@angular/cli'] = '6.0.1';
+      json.devDependencies['@angular/cli'] = '6.1.2';
```

Comment:
Why is this required? Shouldn't the latest migration (nx 6.1 => nx 6.2) do it for us?

Reply:
The CLI _had_ static values for this in their initial 6.0 migration.
* This got rid of the original `root` and `sourceRoot` information so we had no choice but to rely on the key/name of the project.
* It has been fixed in `6.1.2` though so the root is preserved.
  * https://github.com/angular/devkit/pull/985

Now, we can use their new version of the migration to 6.0 in our migration to 6.0 and not overwrite the `root` and `sourceRoot` ourselves.